<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="lizhenzhendeMacBook-Air.local" name="com.qa.tests.GetTest01" tests="4" failures="0" timestamp="07 Mar 2023 04:15:07 GMT" time="1.018" errors="1">
  <testcase name="Test01" time="0.000" classname="com.qa.tests.GetTest01"/>
  <testcase name="Test04" time="1.017" classname="com.qa.tests.GetTest01">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.qa.tests.GetTest01.Test04() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.qa.tests.GetTest01.Test04() didn't finish within the time-out 1000
]]>
    </error>
  </testcase> <!-- Test04 -->
  <testcase name="Test03" time="0.001" classname="com.qa.tests.GetTest01"/>
  <testcase name="Test02" time="0.000" classname="com.qa.tests.GetTest01"/>
</testsuite> <!-- com.qa.tests.GetTest01 -->
